### PR TITLE
Change workspace cloning to be opt-in as well

### LINKED ISSUE #120: Opt-in cloning of main AndcultureCode repos for workspace command
It might be nice to force the user to opt-in to cloning all of the mainline AndcultureCode repos instead of doing it automatically once the base `workspace` command has been run. I didn't realize this was the default behavior, and ended up with a number of cloned repos in a directory that I didn't need them 😛 

We could do this a few ways, like prompting the user to confirm all of the clone actions before continuing when the `workspace` command is run without any flags, or a `-c, --clone` flag with a description about the behavior would work too

Didn't come to mind during the PR @wintondeshong, but what do you think?

### PR DESCRIPTION
**Pull request by wintondeshong**
Fixes #120 

-   [x] Related GitHub issue(s) linked in PR description
-   [x] Destination branch merged, built and tested with your changes
-   [x] Code formatted and follows best practices and patterns
-   [x] Code builds cleanly (no _additional_ warnings or errors)
-   [x] Manually tested
-   [x] Automated tests are passing
-   [x] No _decreases_ in automated test coverage
-   [x] Documentation updated (readme, docs, comments, etc.)
-   [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

File: and-cli-workspace.js [hunk 1]
```diff
@@ -159,27 +163,41 @@ require("./command-runner").run(async () => {
         )
         .parse(process.argv);
 
+    const cloneEnabled = program.clone != null;
+    const forkEnabled = program.fork != null;
+    const usernamesProvided = StringUtils.hasValue(program.usernames);
+
+    // If no flags provided, explain options
+    // -------------------------------------
+    if (!cloneEnabled && !forkEnabled && !usernamesProvided) {
```
Comment (brandongregoryscott) [inline and-cli-workspace.js:172]: I believe we get this 'fall-through' behavior in other commands with something like this, unless we want something a little more in-your-face:

```JS
    if (process.argv.slice(2).length === 0) {
        program.help();
    }
```
(See `and-cli-migration.js` as an example)

We could still keep the variables used to run the individual functions broken out for readability

**Review by brandongregoryscott — APPROVED** [review]
Thanks for adding this so quickly! Left a single refactor suggestion, but I'll leave it up to you if you'd like to keep it as-is.

File: and-cli-workspace.js [hunk 2]
```diff
@@ -159,27 +163,41 @@ require("./command-runner").run(async () => {
...
+        echo.headerError("Atleast one flag is required. See options below...");
```
Comment (HeyKos) [inline and-cli-workspace.js:173]: Nitpick: add space between "At" and "least".

**Review by HeyKos — APPROVED** [review]
Only one nitpick, so it's all good from my end. Thanks for jamming on this.

File: and-cli-workspace.js (hunk 1 above)
Comment (wintondeshong) [reply and-cli-workspace.js]: Check it out again, I wrapped that in a function to make it more readable. Whatcha think?

**Review by brandongregoryscott — APPROVED** [review]
Works for me! I never loved the verbosity of the program arg slicing. We can refactor existing commands as they're updated

**Review by HeyKos — APPROVED** [review]

**Pull request merged**